sourcegraph/sourcegraph
# insights: add site configuration value for commit indexer interval

Closes #22038

### Description

This adds a site configuration value for the commit indexer interval, (in minutes,) and reads the value in when setting the interval in the commit indexer.

### Testing Steps

This was a little tricky to test; it doesn't seem like the site configuration settings get saved through the UI. (Is that expected?) But I did verify:
- Saving an `insights.commit.indexer.interval` value warns that the service will need to be restarted in order for it to take effect.
- If I modify the site config in `dev-private` to include this value, the commit indexer interval is picked up and set appropriately.

Let me know if this sounds reasonable, or if I'm misunderstanding how to test it!
